tflib/imagetest/helm/main.tf

```diff
@@ -1,23 +1,30 @@
 variable "name" {
-  default     = ""
   description = "The name of the helm release. One will be generated if not provided"
+  type        = string
+  default     = ""
 }
 
 variable "namespace" {
   description = "The namespace to install the helm release into."
+  type        = string
```

Comment:
you've already done the work, but in the future you can 🪓 these, they're the default terraform type

Reply:
Happy to axe them, terraformls was yelling at me that there was no type so I just added them in

Reply:
axed